tasks/main.yml

```diff
@@ -1,5 +1,21 @@
 # tasks file
 ---
+- name: Assert keepalived_vrrp_instances is set correctly
+  assert:
+    that:
+      - keepalived_vrrp_instances is defined
+      - keepalived_vrrp_instances is iterable
+    quiet: yes
```

Comment:
Why is this check needed?

Reply:
It's an implementation of "fail fast" strategy so we don't run whole playbook if not all required variables are set. Therefore not strictly needed, but it saves time.